templates/accounts/profile.html

```diff
@@ -3,12 +3,22 @@
 {% block title %}プロフィール閲覧{% endblock %}
 
 {% block content %}
+<p>
   <ul>
     <li>{{user.username }}</li>
     <li>{{user.email }}</li>
     <li>{% if profile.hobby %}{{profile.hobby}}{% else %}趣味　未設定{% endif %}</li>
     <li>{% if profile.introduction %}{{profile.introduction}}{% else %}自己紹介　未設定{% endif %}</li>
   </ul>
   <a href="{% url 'accounts:user_profile_edit' user.pk %}" class="btn btn-primary">プロフィール編集したいなら来な！</a>
-
+</p>
+<p>
+{% for tweet in tweet_list %}
+  <ul>
+    {{tweet.created_at}}
+    {{tweet.user}}
+    {{tweet.contents}}
+  </ul>
+</p>
+{% endfor %}
```

Comment:
```suggestion
<ul>
  {% for tweet in tweet_list %}
  <li>
    {{tweet.created_at}}
    {{tweet.user}}
    {{tweet.contents}}
  </li>
  {% endfor %}
</ul>
```
ulタグ使うならこのような書き方が適切です。見た目の方はbootstrapを用いたりして整えてみてください。できなければbootsrapは使わなくていいです。そもそもul,li使わずにpタグで囲んでみてもいいかもしれませんね。任せます。